CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 1.19.2 (August 26, 2024)
+
+SECURITY:
+
+* ui: Upgrade modules with d3-color as a dependency to address denial of service issue in d3-color < 3.1.0 [[GH-21588](https://github.com/hashicorp/consul/issues/21588)]
+
+IMPROVEMENTS:
+
+* Use Envoy's default for a route's validate_clusters option, which is false. This fixes a case where non-existent clusters could cause a route to no longer route to any of its backends, including existing ones. [[GH-21587](https://github.com/hashicorp/consul/issues/21587)]
+
+BUG FIXES:
+
+* api-gateway: **(Enterprise only)** ensure clusters are properly created for JWT providers with a remote URI for the JWKS endpoint [[GH-21604](https://github.com/hashicorp/consul/issues/21604)]
+
 ## 1.19.0 (June 12, 2024)
```

Comment:
Looks like we're missing 1.19.1. I don't think we usually bother backfilling other lines, but probably ideal to copy in that one here. Guess this means it's missing from `release/1.19.x` too.

Reply:
❤️ thank you.  This has been resolved.  I will open PR to 1.19.x as well.